src/core/actions/index.js

```diff
@@ -1,18 +1,23 @@
 import {
+  LOG_OUT_USER,
   SET_JWT,
   SET_CLIENT_APP,
   SET_LANG,
   ENTITIES_LOADED,
   SET_CURRENT_USER,
 } from 'core/constants';
 
-export function setJWT(token) {
+export function setJwt(token) {
```

Comment:
If we want to change things that's cool but I think most of our acronyms are formatted like this. I know we have a `callApi` too anyway.